### ISSUE TITLE
Todo List

### ISSUE THREAD

**Issue by andrewjong** [andrewjong/Deep-Learning-Paper-Surveys#8]
### ◎ General Deep Learning
- Activation Functions
  - Swish [TO READ]

- Batch Size

- Learning Rate

### ◎ Vision
- Hand-Designed Architectures
  - Accuracy
    - ?
  - Mobile Efficiency
    - MobileNetV1: Depthwise Separable Convolutions
    - MobileNetV2: Inverted and Linear Residual Bottlenecks
    - Fully Learnable Group Convolution for Acceleration of Deep Neural Networks
    - GhostNet
    
  - Squeeze-and-Excitation Networks (seems to be commonly used in a lot of top architectures)

- Neural Architecture Search
  - MNasNet [TO WRITE]
  - Searching for MobileNetV3 [TO READ]
  - EfficientNets [TO WRITE] 
  - Designing Network Design Spaces [TO WRITE]
  - FBNet [TO WRITE]
  - FBNetV2
  
- Data Augmentation
  - AutoAugment [TO READ]
  - Fast AutoAugment [TO READ]
  - AdvProp [TO READ]
  
- Camera Pipeline
  - Deep Demosaicing for Edge Implementation

### ◎ Generative Graphics
- Networks
  - DCGAN [TO WRITE]
  - StyleGAN [TO WRITE]
  - MSG-GAN [TO READ]
  - StyleGAN2 [TO READ]
- Layers
  - Spectral Normalization GAN (SN-GAN) [TO READ]
- Loss Functions
  - WGAN [TO WRITE]
  - WGAN-GP [TO WRITE]
  - Mescheder R1/R2 [TO WRITE]
  - Implicit Competitive Regularization [TO READ]
  
- Systems (fundamental systems that generalize widely)
  - Pix2Pix [TO WRITE]
  - CycleGAN [TO WRITE]
  - Guided Image-to-Image Translation with Bi-Directional Feature Transformation [TO READ]

- Applications
  - Virtual Try-on
    - FW-GAN [TO WRITE]

### ◎ Reinforcement Learning
- Algorithms
  - Q-Learning [TO FIND]
  - PPO [TO FIND]
  
  - World Models [TO READ]
  
- Navigation
  - https://paperswithcode.com/paper/vision-based-navigation-with-language-based [TO READ]

- Model Based

- Model Free
  
### ◎ Natural Language Processing
- Architectures
  - Attention is All You Need (Transformers) [TO READ]
  - BERT [TO READ]
